src/deposit.ts

```diff
@@ -79,9 +84,9 @@ export default async function (zeek?: boolean, l1RpcUrl?: string, l2RpcUrl?: str
       break;
     case "localnet":
       ethProviderUrl =
-        l1RpcUrl == undefined ? "http://localhost:8545" : l1RpcUrl;
+        l1RpcUrl == undefined ? l1RpcUrl : "http://localhost:8545";
```

Comment:
This should be the opposite no? It is using the undefined

Reply:
no, if you don't provide the flag, we'll use the default values. This was actually causing the issue with deposit/withdraws locally